### PR TITLE
BL-1222: Supress record when all its items are inactive.

### PR DESCRIPTION
**Pull request by dkinzer**
* Skips ingesting record when all the items are inactive.

### CODE REVIEW

File: spec/cob_index/macros/custom_spec.rb [hunk 1]
```diff
@@ -1401,6 +1401,47 @@
         expect { subject.map_record(records[14]) }.not_to raise_error
       end
     end
+
+    context "when all items are inactive" do
+      record_text = <<-EOT
+<record>
```
Comment (sensei100) [inline spec/cob_index/macros/custom_spec.rb:1407]: @dkinzer I haven't seen us use this before: record_text = <<-EOT.  What is the benefit of that?  Also, the indentation seems off for the record text fields in all of the new examples.

Comment (dkinzer) [reply spec/cob_index/macros/custom_spec.rb:1407]: @sensei100 it's basically like "  but for situations where you want to use some copy-pasta text (like for a fixture) that can may have a combo of single and double quotes but you don't want to bother figuring out how/where to do the quote escapes.

Comment (dkinzer) [reply spec/cob_index/macros/custom_spec.rb:1407]: Oh, the indentation, ... when you use this quote (I think it's called heredoc?) It's an indicator that the text that follows will  use original format (including spacing).

So another use case is for creating templates.  But basically anything between the heredoc entry and exit codes does not need to be formatted in a specific way.  The linter will know this and not throw an error.

**Comment by sensei100** [comment]
@dkinzer Could it be indented anyway?  I really think it is distracting to have blocks of text that look unformatted.

**Comment by dkinzer** [comment]
@sensei100 let's bring it up in the next DEVOPS meeting.  I prefer to format heredoc text this way to safe time. And it's not without precedent to do it this way.

**Comment by bibliotechy** [comment]
Merging this so we can ship the changes and creating a jira to have a team discussion about heredoc indentation

**Pull request merged**